test/categories/exceptions/unhandled.pls:5:1: ERROR:
Exception: PANIC! (the impossible happened): AAAAAAAA
    Caused by 'Panic'

  | 
5 | raise Panic("AAAAAAAA")
  | ^^^^^^^^^^^^^^^^^^^^^^^